android/build.gradle

```diff
@@ -25,6 +22,7 @@ buildscript {
     dependencies {
         classpath("com.android.tools.build:gradle")
         classpath("com.facebook.react:react-native-gradle-plugin")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin")
```

Comment:
@alexsegura I think it is a duplicate of l.29 just below maybe remove this line?

Reply:
I removed the other line, because it's like this in the [React Native template](https://raw.githubusercontent.com/react-native-community/rn-diff-purge/release/0.73.5/RnDiffApp/android/build.gradle).